UI/remote-text.cpp

```diff
@@ -196,6 +197,10 @@ bool GetRemoteFile(const char *url, std::string &str, std::string &error,
 			}
 		}
 		if (postData) {
+			if (postDataSize > 0)
+				curl_easy_setopt(curl.get(),
+						 CURLOPT_POSTFIELDSIZE,
+						 (long)postDataSize);
 			curl_easy_setopt(curl.get(), CURLOPT_POSTFIELDS,
```

Comment:
Am I reading this right that the first `curl_easy_setopt` is the single statement in the nested `if (postDataSize > 0)` here, and the second `curl_easy_setopt` is outside of the nested `if`?

If so, to avoid confusion, I'd either enclose the statement in curly braces, or at least separate them with a blank line.

Reply:
I put it in braces.